registry.yaml

```diff
@@ -36,39 +36,44 @@ plugins:
       description: Reads Cloudtrail JSON logs from files/S3 and injects as events
       authors: The Falco Authors
       contact: https://falco.org/community
-      url: https://github.com/falcosecurity/plugins/tree/master/plugins/cloudtrail
+      sources: https://github.com/falcosecurity/plugins/tree/master/plugins/cloudtrail
+      url: https://download.falco.org/?prefix=plugins
       license: Apache-2.0
     - id: 3
       source: dummy
       name: dummy
       description: Reference plugin used to document interface
       authors: The Falco Authors
       contact: https://falco.org/community
-      url: https://github.com/falcosecurity/plugins/tree/master/plugins/dummy
+      sources: https://github.com/falcosecurity/plugins/tree/master/plugins/dummy
+      url: https://download.falco.org/?prefix=plugins
       license: Apache-2.0
     - id: 4
       source: dummy_c
       name: dummy_c
       description: Like Dummy, but written in C++
       authors: The Falco Authors
       contact: https://falco.org/community
-      url: https://github.com/falcosecurity/plugins/tree/master/plugins/dummy_c
+      sources: https://github.com/falcosecurity/plugins/tree/master/plugins/dummy_c
+      url: https://download.falco.org/?prefix=plugins
       license: Apache-2.0
     - id: 5
       source: docker
       name: docker
       description: Docker Events
       authors: Thomas Labarussias
       contact: https://github.org/Issif
-      url: https://github.com/Issif/docker-plugin
+      sources: https://github.com/Issif/docker-plugin
+      url: https://download.falco.org/?prefix=plugins
       license: Apache-2.0
     - id: 6
       source: seccompagent
       name: seccompagent
       description: Seccomp Agent Events
       authors: Alban Crequy
       contact: https://github.com/kinvolk/seccompagent
-      url: https://github.com/kinvolk/seccompagent
+      sources: https://github.com/kinvolk/seccompagent
+      url: https://download.falco.org/?prefix=plugins
```

Comment:
```suggestion
```
This plugin is not hosted at download.falco.org

Reply:
> Hey @andreabonanno
> 
> I guess I understood your intent, but I have some concerns. Are we sure the registry is the correct place to store artifacts-related information?
> 
> I thought we would need a separate index file (hosted somewhere, TBD) with all the metadata required to download the plugin files. Moreover, I see `url` as an informative link to the plugin's website (i.e., the project GitHub repository or the plugin website, if any). I don't think that `url` can be used as-is to download plugins. Am I wrong?
> 
> N.B. Not sure how you intend to consume that value, but you cannot use `https://download.falco.org/?prefix=plugins/stable/` (which is an AWS web UI for listing files) as a base URL for downloading files. If you want to use it as a base URL, you should use `https://download.falco.org/plugins/stable/` instead.

Hi and thanks for the feedback!
I do agree that the registry should not directly store artifact-related infos.
I envisioned the `url` field as a _pointer_ to separate index files, possibly hosted somewhere else, and those indexes in turn will be the source of truth for artifact information for that specific plugin. The registry would be the source of truth for those indexes, not for detailed artifacts infos. I can indeed refactor choosing a more expressive name than `url`, and changing the url used right now as a placeholder (as no indexes files present at the moment).